lib/auth/keygen/keygen.go

```diff
@@ -176,6 +176,9 @@ func (k *Keygen) GenerateUserCertWithoutValidation(c services.UserCertParams) ([
 	if c.PermitPortForwarding {
 		cert.Permissions.Extensions[teleport.CertExtensionPermitPortForwarding] = ""
 	}
```

Comment:
This option is also recognized by OpenSSH servers, I think we should only set it if our PortForwarding mode is a global allow